tests/resources/generate-outputs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
```

Comment:
Reference model outputs are now generated within the CI.

Reply:
I'm not sure I'm following why. Isn't the point of these references to be used in regression tests? Or am I missing something here?

Reply:
These we don't use for testing the architecture. It is basically for the infrastructure. That we can load,export etc. That why this is fine in my opinion.

Reply:
ok, fair enough!